### DISCUSSION TITLE
【2023.9.21】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#1117] [meeting]
##### Meeting replay: Bilibili
+ front part：https://www.bilibili.com/video/BV1Zj411k7eR/?spm_id_from=333.999.0.0&vd_source=2aa1b484303c30fb1e5d65c91d790f2d
+ the second half of：https://www.bilibili.com/video/BV1nj411873U/?spm_id_from=333.999.0.0
+ Cluster deployment discussed the previous section：https://www.bilibili.com/video/BV1Zj411k7eR/?spm_id_from=333.999.0.0
+ The end of the clustered deployment discussion：https://www.bilibili.com/video/BV1ip4y1w7Xm/?spm_id_from=333.999.0.0

**Conference link：**
> **Note**: Every two weeks at 9:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

This biweekly will be a long awaited mega-merge and the beginning of our `OpenIM:3.3.0` merge

**Outline of the meeting:**
- [User case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md) and [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by lgz5689** [comment]
introduce uni-app-demo how to run

**Comment by charles-chenzz** [comment]
1. https://github.com/openimsdk/open-im-server/issues/1105
2. https://github.com/openimsdk/open-im-server/issues/1094

better config management for im-server
update the current kickoff logic

**Comment by cubxxw** [comment]
Deployment of openim on Kubernetes with Configuration Management using Helm #1116

**Comment by g6619563** [comment]
Bug: delete a conversation on the web IM, the get_conversations and get_all_conversations interface still returns that data #1111

**Comment by plum330** [comment]
1. What is the usage of operator id in msg?
2. In writing diffusion，Mysql ChatLog Model is using for storing msg, Mongo UserChat model is using for inbox to synchronize msg in V2?
3. In V2 UserChat Model, every mongo docment is appeding to 5000 msg， so how to decide every docment expired time ?
